File: College-Sports-WebApp/ClientApp/src/api/models/last-play.ts
/* tslint:disable */
/* eslint-disable */
import { AthletesInvolved } from '../models/athletes-involved';
import { LastPlayType } from '../models/last-play-type';
import { Probability } from '../models/probability';
export interface LastPlay {
  athletesInvolved?: Array<AthletesInvolved> | null;
  id?: number;
  probability?: Probability;
  scoreValue?: number;
  text?: string | null;
  type?: LastPlayType;
}

File: College-Sports-WebApp/Database/Models/LastPlay.cs
using System.Collections.Generic; 
namespace College_Sports_WebApp.Database.Models{ 

    public class LastPlay
    {
        public string id { get; set; }
        public Type type { get; set; }
        public string text { get; set; }
        public int scoreValue { get; set; }
        public Team team { get; set; }
        public Probability probability { get; set; }
        public List<AthletesInvolved> athletesInvolved { get; set; }
    }

}